src/tests/ftest/dfuse/caching_check.py

```diff
@@ -45,13 +45,16 @@ def test_dfuse_caching_check(self):
 
         self.log_step('Write to the dfuse mount point')
         self.run_ior_with_pool(fail_on_warning=False, stop_dfuse=False)
+        print(self.dfuse.get_stats())
```

Comment:
Basic `print` doesn't show up in logs
```suggestion
        self.log.info(self.dfuse.get_stats())
```

but even better, since `run_remote` already prints the output we can just remove this
```suggestion
```


And the same for below `print`